CHANGELOG.md

```diff
@@ -43,7 +43,8 @@ title: Changelog
 * `<cc-pricing-table>`: add text content to `<cc-button>` elements (toggle State and add buttons) so that they can be identified by assistive technologies.
 * `<cc-zone-input>`: add `tabindex="0"` to the list container so that people can focus and scroll the list without having the change their selection.
 * `resizeObserver`: add `window.requestAnimationFrame` in the resize observer callback to mitigate the `resize observer loop limit exceeded issue`.
-* stories: fix miscellaneous stories not being passed to `enhancedStoryNames`.
+* stories: fix miscalleneous stories not being passed to `enhancedStoryNames`. 
```

Comment:
typo in `miscalleneous`

Reply:
I fixed it but in the wrong commit :P I'll move the fix to the right commit when squashing the feedback commits, thanks !